vertica/datadog_checks/vertica/vertica.py

```diff
@@ -47,30 +44,9 @@ def __init__(self, name, init_config, instances):
 
         self._client_lib_log_level = self.instance.get('client_lib_log_level', self._get_default_client_lib_log_level())
 
-        self._tls_verify = is_affirmative(self.instance.get('tls_verify', False))
+        self._tls_verify = is_affirmative(self.instance.get('tls_verify', True))
```

Comment:
This can be removed, and replaced with the `use_tls` option. off by default.

But to keep backwards compatibility, we should ensure that setting `tls_verify: true` also implicitly set `use_tls: true`, but I think that's already the case with the tls wrapper.